apps/sms/test/marionette/mocks/mock_split_view_mode.js

```diff
@@ -0,0 +1,14 @@
+/* global Services */
+'use strict';
+
+Services.obs.addObserver(function(document) {
+  if (!document || !document.location ||
+      document.location.origin !== 'app://sms.gaiamobile.org') {
+    return;
+  }
+
+  // Switch to split-view mode Inbox instead.
+  if(document.location.pathname === '/index.html') {
```

Comment:
this triggers a full navigation, but "replace" will also replace the current history entry (as opposed to .assign or setting href directly): https://developer.mozilla.org/en-US/docs/Web/API/Location/replace

because of the navigation eveything in the page is stopped and discarded anyway (JS/CSS loading, and even all execution).